src/node.cc

```diff
@@ -3751,7 +3752,9 @@ static void ParseArgs(int* argc,
       default_cipher_list = arg + 18;
     } else if (strncmp(arg, "--use-openssl-ca", 16) == 0) {
       ssl_openssl_cert_store = true;
+      use_ca_flag |= 1;
```

Comment:
See https://github.com/nodejs/node/pull/12087#discussion_r108844523 for some background on this.